Dreamscapes/Ldap/Core/Ldap.php

```diff
@@ -714,7 +719,7 @@ public function startTls()
      */
     public function unbind()
     {
-        ldap_unbind($this->resource);
+        @ldap_unbind($this->resource);
```

Comment:
I think, you can't do anything with LDAP resource whether unbind failed or not. So I personally just ignore anything coming from this method. The only reason I could think of when it would be useful is to catch some logic errors when trying to unbind previously unbinded link. Throwing exception would be too much IMHO, so I would revert the change or leave it - tell me what's yout decision.